contrib/mandoc/config.h

```diff
@@ -13,7 +13,7 @@
 #define HAVE_ENDIAN 0
 #define HAVE_ERR 1
 #define HAVE_FTS 1
-#if defined(__GLIBC__) || defined(__APPLE__)
+#if defined(__linux__) || defined(__APPLE__)
```

Comment:
Is this really #ifndef __FreeBSD__? Or does NetBSD/OpenBSD also have this new const interface?